### PR TITLE
#71 Fixed issue with the unit test for authentication service

### LINKED ISSUE #71: [bug] Fixing issue with unit test on authentication service

### PR DESCRIPTION
**Pull request by Manraj45**
resolves #71

### CODE REVIEW

**Review by nasaku898 — APPROVED** [review]

File: bike-erp/backend/src/test/services/authenticationService/authenticationService.test.ts [hunk 1]
```diff
@@ -46,8 +46,8 @@ describe("Login test", () => {
     //Trying to login with the wrong password
     try {
       await AuthenticationService.login("test@test.com", "tes");
```
Comment (jmezaber) [inline bike-erp/backend/src/test/services/authenticationService/authenticationService.test.ts:48]: the second string is "tes", was it meant to be "test"?

**Review by jmezaber — APPROVED** [review]
Looks good to me!

**Review by huangs08 — APPROVED** [review]
Good job!

File: bike-erp/backend/src/test/services/authenticationService/authenticationService.test.ts (hunk 1 above)
Comment (Manraj45) [reply bike-erp/backend/src/test/services/authenticationService/authenticationService.test.ts:48]: It's just a password. The user password is "test" but I entered "tes" to test a wrong password.

Comment (jmezaber) [reply bike-erp/backend/src/test/services/authenticationService/authenticationService.test.ts:48]: Not a problem at all! Thank you for clarifying!

**Pull request merged**